README.md

```diff
@@ -168,6 +169,22 @@ Success!
 Your keys can be found at: <YOUR_FOLDER_PATH>
 ```
 
+###### `generate-bls-to-execution-change` Arguments 
+
+You can use `bls-to-execution-change --help` to see all arguments. Note that if there are missing arguments that the CLI needs, it will ask you for them.
```

Comment:
If you try to run the `bls-to-execution-change` subcommand with `--help` it will actually skip the help menu and drop right into interactive mode:
```
user@laptop:~/repos/staking-deposit-cli$ python3 ./staking_deposit/deposit.py generate-bls-to-execution-change --help

***Using the tool on an offline and secure device is highly recommended to keep your mnemonic safe.***

Please choose your language ['1. العربية', '2. ελληνικά', '3. English', '4. Français', '5. Bahasa melayu', '6. Italiano', '7. 日本語', '8. 한국어', '9. Português do Brasil', '10. român', '11. Türkçe', '12. 简体中文']:  [English]: 
```